index.html

```diff
@@ -39,16 +39,25 @@
                 };
                 grid.columns[1].renderer = function(cell) {
                     var row = cell.row.data;
-                    cell.element.innerHTML = '<span class="from">' + row.from.firstName + ' ' + row.from.lastName + '</span>';
-                    cell.element.innerHTML += '<span class="subject">' + row.summary + '</span>';
+                    var innerHTML = '<div>';
```

Comment:
Added `<div>` inside cells to make it easier to work with the `display: inline-flex` the new theme is using.